### ISSUE TITLE
Intralobular septal thickening 

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6568]
**Preferred term label:** Intralobular septal thickening 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Thickening of the connective tissue within the secondary lobule

**Parent term (use hpo.jax.org/app)**
Pulmonary interstitial high-resolution computed tomography abnormality HP:0025389

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Idiopathic pulmonary fibrosis ORPHA:2032, OMIM:178500
Sarcoidosis ORPHA:797

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Def: Intralobular septal thickening is a computed tomography finding of increased width of the walls (septa) within a pulmonary lobule. Secondary pulmonary lobules represent a cluster of up to 30 acini 9 supplied by a common distal pulmonary artery and bronchiole. These clustered acini are bounded by interstitial fibrous septa (interlobular septa) which outline an irregular polyhedron of varying size between 1 and 2.5 cm. Interlobular septal thickening is seen on chest radiographs as thin linear opacities at right angles to and in contact with the lateral pleural surfaces near the lung bases. In contrast, intralobular septal thickening are visible as fine linear opacities in a lobule when the intralobular interstitial tissue is abnormally thickened. When nu
merous, they may appear as a fine reticular pattern. 

Comment: Intralobular lines may be seen in various conditions, including interstitial fibrosis and alveolar proteinosis.
PMID: 18195376  (Fleischner society guidelines)
Also
https://pubmed.ncbi.nlm.nih.gov/27852954/

@eswietlik  -- please check this definition, which is intended to provide context for those who are not specialists. 
Also, according to Fleischner society guidelines, this finding appears to be called Intralobular lines (https://pubmed.ncbi.nlm.nih.gov/18195376/ see Figure 31). Should we use that as the primary label or as a synonym?

**Comment by pnrobinson** [comment]
added
